frontend/src/components/auth/Registration.jsx

```diff
@@ -5,31 +5,43 @@ import { Link, useNavigate } from "react-router-dom";
 export default function Registration() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  // Set useState for isTeacher with a default value of false
+  const [isTeacher, setIsTeacher] = useState(false);
   // Removed useState for name and added for firstName, lastName
   const [firstName, setFirstName] = useState("");
   const [lastName, setLastName] = useState("");
   const [registrationMessages, setRegistrationMessages] = useState("");
   const navigate = useNavigate();
-  
-
-
   const [userData, setUserData] = useState({});
-
   const registrationUrl = `${API_URL2}/signup`;
+
+  // Toggles the isTeacher state and logs to console
+  const handleClick = () => {
+    setIsTeacher((prevIsTeacher) => {
+      const newIsTeacher = !prevIsTeacher;
+      if (newIsTeacher) {
+        console.log("I am a Teacher");
+      } else {
+        console.log("I am not a teacher");
+      }
+      return newIsTeacher;
+    });
```

Comment:
1. In general, we want to avoid committing `console.log` lines.
2. Without the console.log, there's no reason to create that variable
```suggestion
    setIsTeacher((prevIsTeacher) => !prevIsTeacher });
```

Reply:
I am sorry. I was just trying to see if it worked without having my backend connected yet. I took the front end ticket first in case someone wanted to take the backend one, and I wanted to test it. Should I remove it?